### PR TITLE
Issue 9 add UMass Memorial Marlborough.

### PR DESCRIPTION
**Pull request by benpatterson**
Addresses #9, leveraging the MyChart.js implementation.

Also see https://github.com/adpjay/covid-vaccine-scrapers/pull/1 for earlier version.

CC @adpjay @livgust 

Still haven't seen a `true` availability, FTR.

### CODE REVIEW

File: site-scrapers/UMassMemorialMarlborough.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    // Setup the return object.
+    const results = { availability: {}, hasAvailability: false };
+
+    // Setup the post data to answer the survey questions
+    let startDate = new Date();
+    startDate.setDate(startDate.getDate());
+    const filters =
+        '{"Providers":{"56394":true,"56395":true,"56396":true,"56475":true,"56476":true,"56526":true,"56527":true,"56528":true,"56529":true,"56530":true,"56531":true,"56532":true,"56533":true},"Departments":{"102001144":true,"104001144":true,"111029146":true},"DaysOfWeek":{"0":true,"1":true,"2":true,"3":true,"4":true,"5":true,"6":true},"TimesOfDay":"both"}';
+    // The data can only be returned for one week. Check the calendar for 10 future weeks.
+    for (const i of Array(10).keys()) {
```
Comment (adpjay) [inline site-scrapers/UMassMemorialMarlborough.js:33]: There is now an `AddFutureWeeks` function that will handle this loop for you. You just need to provide the url information and a callback function to generate the formatted postdata.

**Review by adpjay — CHANGES_REQUESTED** [review]

**Review by livgust — CHANGES_REQUESTED** [review]
@benpatterson please include the change requested by @adpjay
Also, MyChartAPI was moved to a subfolder so that will need to be updated.

File: site-scrapers/UMassMemorialMarlborough.js (hunk 1 above)
Comment (benpatterson) [reply site-scrapers/UMassMemorialMarlborough.js]: done

**Comment by benpatterson** [comment]
I have refactored based on recent changes. Let me know what you think.

**Review by adpjay — APPROVED** [review]
Looks good! Thanks @benpatterson !

**Review by adpjay — APPROVED** [review]
Just adding this again as github says it is awaiting my approval.

**Comment by benpatterson** [comment]
Ready for ya @livgust

**Review by livgust — APPROVED** [review]
Verified that output looks good (no availability, matches what I see online); I'll add the new `timestamp` directly in `master`

**Pull request merged**